src/components/Item/VisualizationItem/Item.js

```diff
@@ -229,7 +229,7 @@ export class Item extends Component {
         const itemName = pluginManager.getName(item);
 
         return (
-            <div style={{ display: 'flex', alignItems: 'center' }}>
+            <div style={{ display: 'flex' }}>
```

Comment:
Is this on purpose?
I think I added it for centering SV within the dashboard item.

Reply:
From what I can see it only affects the title row.